evap/results/templates/average_grade_badge.html

```diff
@@ -0,0 +1,16 @@
+{% load results_templatetags %}
+{% load evaluation_filters %}
+
+{% spaceless %}
+{% if show_grades and average %}
+    <div class="grade-bg-result-bar text-center{% if participants_warning %} participants-warning{% endif %}" style="background-color: {{ average|gradecolor }}">
+        {% if result.question.is_yes_no_question %}
+            {{ result.approval_count|percentage:result.total_count }}
+        {% else %}
+            {{ average|floatformat:1 }}
+        {% endif %}
+    </div>
+{% else %}
+    <div class="grade-bg-result-bar text-center grade-bg-disabled"><span class="far fa-eye-slash"></span></div>
+{% endif %}
+{% endspaceless %}
```

Comment:
perhaps consider making two versions of this. one for results index and the overview part of results course detail, because those don't have`participants_warning` and no `is_yes_no_question`, and the other one for the results bar, and in there you're then free to put the tooltip wherever necessary ;)